### ISSUE TITLE
[HOLD for payment + bonus] Make bank account loading animations consistently top docked in the right hand pane 

### ISSUE THREAD

**Issue by MitchExpensify** [Expensify/App#4696]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Open New Expensify 
2. Click "+"
3. Click "Create Workspace"
4. Click "Get Started"

## Expected Result:
We expect **all** loading animations in the bank account connection flow to be positioned at the top of right hand pane here:

![image](https://user-images.githubusercontent.com/36425901/129764566-52ff82e0-ca1d-440e-921a-6023592829ea.png)

## Actual Result:
Some loading animations are positioned in the center here (Validate step):
![image](https://user-images.githubusercontent.com/36425901/129764971-94c359bf-b38b-4ba3-86eb-3ccab90ad4fd.png)

And here (clicking Get Started)
![image](https://user-images.githubusercontent.com/36425901/129765139-0bbf4fd2-2eb2-495b-a0c9-f3c34b601fc7.png)

https://user-images.githubusercontent.com/36425901/129765531-565686a8-546e-4771-b682-87a702b8c08b.mov

## Workaround:
Simply ignore, this is a design change.

## Platform:

Where is this issue occurring?

- Web
- iOS
- Android
- Desktop App
- Mobile Web

**Version Number:** v1.0.85-9
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation
**Expensify/Expensify Issue URL:**

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by MelvinBot** [comment]
Triggered auto assignment to @arielgreen (`AutoAssignerTriage`), see https://stackoverflow.com/c/expensify/questions/4749 for more details.

**Comment by MitchExpensify** [comment]
Thinking this is a good external candidate issue

**Comment by Santhosh-Sellavel** [comment]
## Proposal
We do pass the below styles to that component for the loading animation view

https://github.com/Expensify/App/blob/6b3a938b3074ec64816ea5188683f383171543c9/src/styles/styles.js#L1789-L1796

We could remove the` justifyContent: 'center'` that solves the issue.

**Comment by Santhosh-Sellavel** [comment]
We should create a new copy of that style. Because it was used by common `FullScreenLoadingIndicator` also!

**Comment by Santhosh-Sellavel** [comment]
<img width="221" alt="Screenshot 2021-08-17 at 11 04 44 PM" src="https://user-images.githubusercontent.com/85645967/129773906-cb11202e-1ef8-421b-bf64-c2a7145eff38.png">

Also, I noticed in the expected Image there is a nav header with a close option it's not available currently will add that post discussion!

Also, a different message is shown, will update that too based on the input. Thanks!

**Comment by MelvinBot** [comment]
Triggered auto assignment to @TomatoToaster (`Engineering`), see https://stackoverflow.com/c/expensify/questions/4319 for more details.

**Comment by MelvinBot** [comment]
Triggered auto assignment to @kadiealexander (`External`), see https://stackoverflow.com/c/expensify/questions/8582 for more details.

**Comment by TomatoToaster** [comment]
Applying the label so a posting can be made. Agreed this can be external (and there's a proposal a few comments up)

**Comment by kadiealexander** [comment]
Upwork post: https://www.upwork.com/jobs/~0176449bcbd8e3cd2c

**Comment by MelvinBot** [comment]
Triggered auto assignment to @Jag96 (`Exported`), see https://stackoverflow.com/c/expensify/questions/7972 for more details.

**Comment by kadiealexander** [comment]
Still waiting on proposals, double-checked that the links were still in the job post (they're still good).

**Comment by kadiealexander** [comment]
@Jag96 when you get a moment could you please take a look at @Santhosh-Sellavel's proposal above?

**Comment by Jag96** [comment]
@Santhosh-Sellavel your updated proposal looks good on web, one thing I noticed on mobile is that the indicator is pretty close to the top, maybe we can add update the style to ensure it looks good on mobile as well? Tagging @Expensify/design to confirm we'd want to add more space:

<img src='https://user-images.githubusercontent.com/3981102/130495969-0a15479f-aa21-4644-977a-4d0be23e090c.png' width='400'/>

> Also, I noticed in the expected Image there is a nav header with a close option it's not available currently will add that post discussion!

@MitchExpensify is this something we want to add to the steps in the beginning? I thought we only had these for certain steps in the flow, but not sure. If we end up wanting to update every step to show the title while the loading screen is active I think we should create a new issue, since that increases the scope of work here.

@Santhosh-Sellavel also could you apply to the Upwork job so once design clarifies we can accept your proposal and get an offer sent over?

**Comment by Santhosh-Sellavel** [comment]
> <img alt="Screenshot 2021-08-17 at 11 04 44 PM" width="221" src="https://user-images.githubusercontent.com/85645967/129773906-cb11202e-1ef8-421b-bf64-c2a7145eff38.png">
> 
> Also, I noticed in the expected Image there is a nav header with a close option it's not available currently will add that post discussion!
> 
> Also, a different message is shown, will update that too based on the input. Thanks!

@Jag96 Any thoughts on this? Can we skip it now?

**Comment by Santhosh-Sellavel** [comment]
@kadiealexander @Jag96  Applied for the job on Upwork, thanks!

**Comment by Jag96** [comment]
I think we can skip updating the Nav Header for this issue, I will defer to @MitchExpensify about the copy change and I'll defer to @Expensify/design regarding the spacing on the top on mobile. Once we get clarification on those 2 items you're good to move forward with a PR. cc @kadiealexander

**Comment by shawnborton** [comment]
In terms of spacing, I would expect this view to look like this where we have a header above the empty state:
![image](https://user-images.githubusercontent.com/2319350/130578768-9de38588-c034-456d-bd5d-83947b3f445e.png)

**Comment by Santhosh-Sellavel** [comment]
cc: @shawnborton @Jag96 
We have a header height defined as `65` here
https://github.com/Expensify/App/blob/95bf36d950f57d01ae37af2e0d8f2e664a273b75/src/styles/variables.js#L2
So we can just add a `marginTop: variables.contentHeaderHeight ` to the below style which proposed earlier.
> <img width="693" alt="Screenshot 2021-08-17 at 10 48 10 PM" src="https://user-images.githubusercontent.com/85645967/129771673-e1202dc2-74a9-4cf4-8c3f-6f6f0a68931d.png">

This is how it looks, 
### iOS
![Simulator Screen Shot - iPhone 12 - 2021-08-24 at 20 14 05](https://user-images.githubusercontent.com/85645967/130638464-70a117b1-7784-4076-b503-89fa4b316579.png)

### Web

https://user-images.githubusercontent.com/85645967/130638916-d3298848-68ff-4d56-838a-4c0c31af6023.mov

**Comment by shawnborton** [comment]
Apologies for the confusion here, but why can't we use a header here? I would say that if there is no actual header that is shown, we just center the content in the view. But that being said, I don't particularly like the idea of showing this view without a header, which means there is no way to exit if things get stuck.

**Comment by Santhosh-Sellavel** [comment]
You are suggesting a header with just a close button, no title?

**Comment by Santhosh-Sellavel** [comment]
cc: @shawnborton 
Something Like this?
https://user-images.githubusercontent.com/85645967/130649150-bd0dade3-f52e-4210-b02d-6bab2d8cd48c.mov

**Comment by shawnborton** [comment]
Yeah, so then if you can add a small "x" then let's just add the full header that includes a title please. Thanks!

**Comment by Jag96** [comment]
Great! Sounds like we've got it sorted, @Santhosh-Sellavel I've assigned you and sent the offer in Upwork, you're good to start on a PR

**Comment by Santhosh-Sellavel** [comment]
Need title copy and new message copy @Jag96  with translations

**Comment by MitchExpensify** [comment]
Title: "One Moment" 

Message: "We are taking a look at your information. You will be able to continue with next steps shortly."

**Comment by Jag96** [comment]
@MitchExpensify so currently we are showing that on the VBA loading screen, I think we need the content for what the header should be, similar to the `Validate` header:

| Validate Header | Need Header value |
|---|---|
| ![image](https://user-images.githubusercontent.com/3981102/130652148-91ac50ae-24d2-4d7b-b3eb-765ae4be2448.png) | ![image](https://user-images.githubusercontent.com/3981102/130652160-436c65a9-4138-4807-b238-3895fc4dff25.png) |

Should the header value be whatever step the user just filled out? So after they hit `Submit` on the `Add Bank Account` screen, should the value on the loading screen be `Add Bank Account`? Or do we want to just have it be a static header title like `One Moment`?

**Comment by MitchExpensify** [comment]
I think a static "One Moment" header is fine personally

**Comment by Santhosh-Sellavel** [comment]
@MitchExpensify like this?
![Simulator Screen Shot - iPhone 12 - 2021-08-24 at 21 50 39](https://user-images.githubusercontent.com/85645967/130653289-bfd10b59-701e-4323-a211-bbad3d746aa6.png)

**Comment by MitchExpensify** [comment]
That works for me @Santhosh-Sellavel 

 I notice the Validate message is left aligned and the One Moment message is center aligned. Which should we commit to @shawnborton ?

**Comment by shawnborton** [comment]
Hmm maybe centered looks nicer under the graphic?

**Comment by Santhosh-Sellavel** [comment]
Are we all good with this @shawnborton @Jag96 @MitchExpensify 
https://github.com/Expensify/App/issues/4696#issuecomment-904787489

**Comment by Jag96** [comment]
Looks good to me!

**Comment by Santhosh-Sellavel** [comment]
These are existing translations:

```
oneMoment: 'Un momento...',
oneMoment: 'One moment...',
```
Can update I like this,
oneMoment: 'One Moment', // en
oneMoment: 'Un Momento', // es

Are the above translations okay?
Let `Moment` or `moment`?

@Jag96 @MitchExpensify

**Comment by Jag96** [comment]
If those are no longer used then I think you're good to update them. I'd go with `One Moment` since we follow that capitalization in a few places (ex - `Change Password`)

**Comment by Santhosh-Sellavel** [comment]
@Jag96 VBA Animation is broken in android
Can I post a separate issue for this?

https://user-images.githubusercontent.com/85645967/130666650-8cd68c50-4932-48d7-8af6-3940ab3015fe.mov

**Comment by Santhosh-Sellavel** [comment]
@Jag96 need to update the fresco version here to 2.5.0
https://github.com/Expensify/App/blob/396e320692d0be6420b23bbe4d234e36560630aa/android/app/build.gradle#L241-L242

**Comment by Jag96** [comment]
@Santhosh-Sellavel good catch, can you update the fresco version as part of https://github.com/Expensify/App/pull/4805?

**Comment by Santhosh-Sellavel** [comment]
Sure @Jag96!

**Comment by Santhosh-Sellavel** [comment]
@Jag96 Will get a bonus for this issue 😂 ✌️?

**Comment by Jag96** [comment]
@Santhosh-Sellavel yes we can add a bonus for the extra work you did on this one 👍

**Comment by shawnborton** [comment]
Looks good to me 👍

**Comment by kadiealexander** [comment]
Paid with bonus today (7 days after deployment to production).
